test/path_resolver_test.cr

```diff
@@ -74,9 +74,9 @@ module Shards
       end
     end
 
-    private def resolver(name, config = {} of String => String)
-      config["path"] = git_path(name)
-      dependency = Dependency.new(name, config)
+    private def resolver(name, **config)
+      config = config.merge(resolver: "path", url: git_path(name))
+      dependency = Dependency.new(name, **config)
       PathResolver.new(dependency)
     end
```

Comment:
Same here no need for `config`. Passing anything but `path` doesn't even make sense for `PathResolver` anyway.